File: samples/conference/registration-bc/src/main/java/com/microsoft/conference/registration/commands/Orders/PlaceOrder.cs
using System;
using System.Collections.Generic;
using System.Collections.ObjectModel;
using ENode.Commanding;

namespace Registration.Commands.Orders
{
    public class PlaceOrder : Command<Guid>
    {
        public PlaceOrder() : base(Guid.NewGuid())
        {
            Seats = new Collection<SeatInfo>();
        }

        public Guid ConferenceId { get; set; }
        public IList<SeatInfo> Seats { get; set; }
    }
}

File: samples/conference/common/src/main/java/com/microsoft/conference/common/registration/commands/Orders/PlaceOrder.java
package com.microsoft.conference.common.registration.commands.Orders;

import org.enodeframework.commanding.Command;
import org.enodeframework.common.utilities.ObjectId;

import java.util.ArrayList;
import java.util.List;

public class PlaceOrder extends Command<String> {
    public String ConferenceId;
    public List<SeatInfo> Seats;

    public PlaceOrder() {
        super(ObjectId.generateNewStringId());
        Seats = new ArrayList<>();
    }
}
